lib/solana.cpp

```diff
@@ -181,6 +193,29 @@ void to_json(json &j, const commitmentconfig &config) {
   }
 }
 
+void to_json(json &j, const mintOrProgramIdConfig &config) {
```

Comment:
should be if else right, and may be should be an like

Reply:
@godmodegalactus , Thanks. Yes it is either mint or programId ! I have fixed it.